driver/src/orderbook/filtered_orderbook.rs

```diff
@@ -7,12 +7,24 @@ use serde::Deserialize;
 use std::collections::{HashMap, HashSet};
 use std::str::FromStr;
 
+#[derive(Clone, Debug, Deserialize, PartialEq, Eq)]
+enum TokenFilter {
+    Whitelist(HashSet<u16>),
+    Blacklist(HashSet<u16>),
+}
+
+impl Default for TokenFilter {
+    fn default() -> Self {
+        TokenFilter::Whitelist(HashSet::new())
```

Comment:
I think the default behaviour should be to consider all tokens (i.e. an empty black-list).